### PR TITLE
docs: Update formatting, ensure pages render properly

### PR DESCRIPTION
**Pull request by ejlee3**
## PR Checklist

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
- [x] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [ ] Updated Documentation as relevant to the changes
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## Issue this PR will close

close: #**issue_number**

## Test Instructions if applicable

run `make serve-docs` and ensure pages render properly

### CODE REVIEW

File: docs_src/OVMS/quick_stream_density.md [hunk 1]
```diff
@@ -20,23 +20,29 @@ To stop all running pipelines and clean up, run
...
+        ```bash
+        cd ./benchmark-scripts
+        make build-benchmark
+        make build-xpu
+        ```
+
+        !!! Success
+            Run `docker images` to ensure that both the `benchmark:dev` and `benchmark:xpu` images were built.
+
+!!! Warning
```
Comment (jim-wang-intel) [inline docs_src/OVMS/quick_stream_density.md:44]: should this be indented to align above formatting issue?

File: docs_src/OVMS/pipelinebenchmarking.md [hunk 2]
```diff
@@ -153,19 +153,22 @@ The primary purpose of benchmarking with a specified number of pipelines is to d
...
-    PIPELINE_PROFILE="grpc_python" sudo -E ./benchmark.sh --stream_density 14.9 --logdir mytest/data --duration 60 --init_duration 60 --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0
-    ```
+    === "Stream density benchmark script in python"
+
+        ```bash
+        PIPELINE_PROFILE="grpc_python" sudo -E ./benchmark.sh --stream_density 14.9 --logdir mytest/data --duration 60 --init_duration 60 --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0
+        ```
 
-If prefix is not provided, then the default value is "grpc_python".
+    !!! Note
```
Comment (jim-wang-intel) [inline docs_src/OVMS/pipelinebenchmarking.md:170]: is this indentation correct with !!!Example one above or this is a nested one?

Comment (ejlee3) [reply docs_src/OVMS/pipelinebenchmarking.md:170]: I was thinking this would be nested into the !!! Example box, but outside of the tabs.

**Review by jim-wang-intel — APPROVED** [review]

**Review by antoniomtz — APPROVED** [review]

File: docs_src/dev-tools/overview.md [hunk 3]
```diff
@@ -1,2 +1,3 @@
 # Developer Overview
 
+Developers can learn more about specific tools available to assist in both running pipelines and benchmarking.
```
Comment (NeethuES-intel) [inline docs_src/dev-tools/overview.md:3]: can you add about documentation tool also as it is covered under this section ? eg
```suggestion
Developers can learn more about specific tools available to assist in running pipelines, benchmarking and documentation.
```

**Review by NeethuES-intel — COMMENTED** [review]
Added one comment

**Pull request merged**